app/resources/view.html.template

```diff
@@ -265,7 +264,7 @@
 
             {% if linked_person_info %}
               <div class="other section">
-                <h3>{% trans "Possible duplicates" %}</h3>
+                <h2>{% trans "Possible Duplicates" %}</h2>
```

Comment:
I thought it looked nicer in headers. I didn't realize there would be translation implications. I'll revert them but I do think it's something to consider.